### DISCUSSION TITLE
Tax management

### DISCUSSION THREAD

**Discussion by rrrnld** [saleor/saleor#9276] [Q&A]
Hi! I'm currently evaluating Saleor for a company that would like to sell books primarily, physical and digital, worldwide. I have some questions about how I would go about setting up the store.

- How exactly are taxes going to be managed? What exactly is the purpose of Vatlayer and Avalara? I see they have plugins coming by default with Saleor 3.1 but I'm unsure what their purpose is. Do I need them in any case?
- I tried setting up Vatlayer and  have some books get taxed accordingly. I was able to fetch tax data in the Settings → Taxes area, but no matter what I come up with to enter into the "taxes" field for the product type, I don't get any result in the dashboard. Why is that? What should I enter there?

Thanks for any help and advice!

**Comment by korycins** [comment]
Currently, to have any support for taxes on the Saleor side you have to activate one of the built-in plugins or build one on your own. This will be changed in feature releases but I don't have any ETA. 
Vatlayer works for EU, activating them and assigning tax category should include taxes in checkout for any products that are sold in EU.
Avalara is a more advanced tool, it can calculate taxes for each country. Taxes are included during a checkout process, when all user's details are provided.
